snowshu/samplings/sample_sizes/brute_force_sample_size.py

```diff
@@ -9,24 +11,23 @@ class BruteForceSampleSize(BaseSampleSize):
     """
 
     def __init__(self,
-                 percentage:float):
-
-        self._percentage=percentage
+                 percentage: float):
+        self._percentage = percentage
 
     @property
-    def percentage(self)->float:
+    def percentage(self) -> float:
         return self._percentage
 
     @percentage.setter
-    def percentage(self,val:float)->None:
+    def percentage(self, val: float) -> None:
         """validates percentage between 1 and 99% before setting."""
-        if (0.01 <= val <= 0.99):
-            self._percentage=val
+        if 0.01 <= val <= 0.99:
+            self._percentage = val
         else:
             raise ValueError(f"Percentage must be between 0.01 and 0.99, is {val}")
 
 
-    def size(self,population:int)->int:
+    def size(self, population: int) -> int: # noqa pylint: invalid-overridden-method, arguments-differ
```

Comment:
RN `size` is a property of the parent class [`BaseSampleSize`](https://github.com/Health-Union/snowshu/pull/57/files#diff-6152cfa247fd7c24c569d06b20cb582f0aeff0b2873e29542d322ff843656a4dR6) but actually this should a property of the class [`BaseSampling`](https://github.com/Health-Union/snowshu/pull/57/files#diff-8e282c390a4c1ba71fd768e192d9940dd71ad83cc8f62ff99f105e0ee1a13b51R53)